### DISCUSSION TITLE
Removal of {{LearnBox}} macro

### DISCUSSION THREAD

**Discussion by teoli2003** [mdn/mdn-community#65] [Localization] [answered]
Hey @mdn/localization-team-leads, 

The `{{LearnBox}}` macro is being [removed from mdn/content](https://github.com/mdn/content/pull/15738) and [deprecated in mdn/yari](https://github.com/mdn/yari/pull/6184). 

When you have a moment, we would appreciate it if you could start the process of removing the macro from your area of `mdn/translated-content`.

Thank you for your continued work to ensure the health of our translated content.

I found 19 occurrences left on `mdn/translated-content`:
```
> rg LearnBox -l
files/de/glossary/index.html
files/de/learn/index.html

files/es/glossary/index.html
files/es/learn/index.html

files/fr/glossary/index.md
files/fr/learn/index.md

files/ja/glossary/index.html
files/ja/learn/index.html

files/ko/learn/index.html

files/pl/glossary/index.html
files/pl/learn/index.html

files/pt-br/glossary/index.html
files/pt-br/learn/index.html

files/ru/glossary/index.html
files/ru/learn/index.html

files/zh-cn/glossary/index.md
files/zh-cn/learn/index.md

files/zh-tw/glossary/index.html
files/zh-tw/learn/index.html
```

**Comment by SphinxKnight** [comment]
https://github.com/mdn/translated-content/pull/5469 it is for `fr`

**Comment by mfuji09** [comment]
I have made pull requests for `ja`:
- https://github.com/mdn/translated-content/pull/5472
- https://github.com/mdn/translated-content/pull/5473

**Comment by yin1999** [comment]
I have created 3 pr for `l10n-zh`:

- mdn/translated-content#5480
- mdn/translated-content#5479
- mdn/translated-content#5478

**Comment by schalkneethling** [comment]
Thank you so much for attending to this so quickly and efficiently everyone ❤️

**Comment by alattalatta** [comment]
Done in `ko`: https://github.com/mdn/translated-content/pull/5459

On a sidenote, I think it'll help to have these localization discussions labeled with locales, like how it is in translated-content: https://github.com/mdn/translated-content/issues/5170

**Reply by teoli2003** [reply]
This is a good idea. Unfortunately, I can't set labels here.

**Reply by SphinxKnight** [reply]
Seconding @alattalatta, I think this can of topic may be filed as an issue over mdn/translated-content and be tracked there.

**Reply by teoli2003** [reply]
So you would like a single issue in mdn/translated-content for all languages or one for each language? I can do either (and still post here)

**Reply by SphinxKnight** [reply]
One for all language should be enough in my opinion, it's then up to each locale's maintainers to divide it if necessary. Using labels there will make it easier to see the remaining locales.

**Reply by schalkneethling** [reply]
I have added the appropriate labels.

**Reply by teoli2003** [reply]
I have a question: what is the purpose of listing these removals here, wouldn't a simple issue under mdn/translated-content be enough?

**Reply by SphinxKnight** [reply]
To me, the most important thing is to have the 'signal' / 'notification' somewhere for this kind of removal. This sort of communication has greatly improved in the last weeks :) 

After that, it boils down to tracking this in task management, which at the moment is done via mdn/translated-content (which could be tagged as 'community' in itself 😁)

**Reply by teoli2003** [reply]
Yes, the communication improvement is really noticeable and great! More of it!

**Comment by lex111** [comment]
Done it for Ru locale https://github.com/mdn/translated-content/pull/5570

**Comment by SphinxKnight** [comment]
Here are the 8 remaining files with this one:

- inactive locales (`de`/`pl`) (I'll take care of them)
  - `de\glossary\index.html`
  - `de\learn\index.html`
  - `pl\glossary\index.html`
  - `pl\learn\index.html`
- @mdn/yari-content-es 
  - `es\glossary\index.html`
  - `es\learn\index.html`
- @mdn/yari-content-pt-br 
  - `pt-br\glossary\index.html`
  - `pt-br\learn\index.html`

**Reply by SphinxKnight** [reply]
https://github.com/mdn/translated-content/pull/6325 for `de` and `pl`

**Comment by JuanVqz** [comment]
Opened PR for @mdn/yari-content-es  :rocket: 
https://github.com/mdn/translated-content/pull/6329

**Reply by Graywolf9** [reply]
Merged! Thank you so much!

**Comment by carlossgabriel** [comment]
Opened PR for @mdn/yari-content-pt-br 👨🏻‍💻
[mdn/translated-content#6567](https://github.com/mdn/translated-content/pull/6567#issue-1288007674)

**Reply by teoli2003** [reply]
Thanks a lot for this PR, @carlossgabriel. Do you think you can create a PR for the pt-BR to solve this one, too: https://github.com/mdn/mdn-community/discussions/67 It would be helpful (only 2 occurrences left in pt-BR, and pretty straightforward to solve).

Thank you!

**Comment by teoli2003** [comment, marked as answer]
I ran `rg LearnBox`, and I can claim that all occurrences of `{{LearnBox}}` have been removed.

Thank you very much, everybody! Well done!
